initramfs-init.in

```diff
@@ -364,6 +364,11 @@ if [ "${s390x_net%%,*}" = "qeth_l2" ]; then
 	echo 1 > /sys/bus/ccwgroup/drivers/qeth/"${_channel%%,*}"/online
 fi
 
+if [ -n "$ssh_install" ]; then
```

Comment:
How about we drop the `ssh_install` and instead do:
````sh
if [ -n "$ssh_key" ] || [ -n "$ssh_pass" ]; then
    ...
````

Reply:
Fixed.